pkg/monitortests/kubeapiserver/auditloganalyzer/audit_log_summarizer.go

```diff
@@ -468,8 +490,14 @@ func writeAuditLogDL(artifactDir, timeSuffix string, auditLogSummary *AuditLogSu
 		for rk, rv := range uv.perResourceRequestCount {
 			for vk, vv := range rv.perVerbRequestCount {
 				for sk, sv := range vv.perHTTPStatusRequestCount {
-					rows = append(rows, map[string]string{"User": uv.user, "Resource": rk.Resource, "Verb": vk, "HttpStatus": strconv.FormatInt(int64(sk), 10), "RequestCount": strconv.FormatInt(int64(sv), 10)})
+					user, unmodifiedUser := cleanupUser(uv.user)
+					data := map[string]string{"User": user, "Resource": rk.Resource, "Verb": vk, "HttpStatus": strconv.FormatInt(int64(sk), 10), "RequestCount": strconv.FormatInt(int64(sv), 10)}
+					if len(unmodifiedUser) > 0 {
+						data["unmodified_user"] = unmodifiedUser
```

Comment:
If we have modified the user the unmodified value will be added to the outupt data.  It isn't added to the schema so it will not be loaded in the bigquery table but we can do that if we want.

Reply:
Produces
```    
{
            "HttpStatus": "200",
            "RequestCount": "2",
            "Resource": "runtimeclasses",
            "User": "system:node-0",
            "UserUnmodified": "system:node:ci-op-6cqn1jmf-03fd1-5qt7q-worker-a-srqxc",
            "Verb": "list"
 },
```